scheduler/system_sched.go

```diff
@@ -390,3 +394,28 @@ func (s *SystemScheduler) computePlacements(place []allocTuple) error {
 
 	return nil
 }
+
+// addBlocked creates a new blocked eval for this job on this node
+// - Keep blocked evals in the scheduler, but just for grins
```

Comment:
Not sure I understand this comment?  why do we do it for grins?  Is this following a pattern used in non system jobs?  I'd suggest avoiding if it's not needed.